cwf/gateway/helm/cwf/templates/bin/_install_gateway.sh.tpl

```diff
@@ -91,8 +91,9 @@ fi
 
 if [ "$GW_TYPE" == "$XWF" ]; then
   MODULE_DIR="xwf"
+  CONNECTION_MODE=${MODE:=tcp}
   ANSIBLE_CONFIG="$INSTALL_DIR"/magma/"$MODULE_DIR"/gateway/ansible.cfg \
-    ansible-playbook -e "xwf_ctrl_ip=$XWF_CTRL" \
+    ansible-playbook -e "xwf_ctrl_ip=$XWF_CTRL connection_mode=$CONNECTION_MODE" \
```

Comment:
I'm not sure that we are using this script any more, maybe we should remove it from here